### PR TITLE
release: 0.9.0

### PR DESCRIPTION
**Pull request by mdonadoni**

### CODE REVIEW

File: templates/page.html [hunk 1]
```diff
@@ -432,10 +432,11 @@ <h3>Near-term</h3>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">
-              <a href="https://github.com/orgs/reanahub/projects/50" class="card-link">Workspace file retention</a>
+              <a href="https://github.com/reanahub/reana-client-go" class="card-link">Go client and stable API</a>
```
Comment (mdonadoni) [inline templates/page.html:435]: Should I remove the link or point to something else?

Comment (tiborsimko) [reply templates/page.html:435]: We usually link to Kanban boards there, but we don't have one yet... I guess we can keep the link to the Go client repository.

**Review by tiborsimko — APPROVED** [review]
Looks nice :+1: 

The CI complains about a link that should be fixed after we release reana 0.9.0

File: templates/page.html [hunk 2]
```diff
@@ -432,10 +432,11 @@ <h3>Near-term</h3>
...
             </h5>
             <p class="card-text">
-                Introduce user-configurable workspace file retention policy allowing to specify that some files are to be deleted automatically 1 day after workflow finishes, other files retained for 356 days, etc. <a href="https://github.com/reanahub/reana-client/issues/610">[rfc]</a>
+              Introduce new command-line client written in Go, to improve performance and ease of use. At the same time, stabilize the current REST API to guarantee compatibility between clients and REANA.
```
Comment (tiborsimko) [inline templates/page.html]: At the same time... -> We could perhaps shorten to: "At the same time, stabilise the REST API versioning for future evolution."

**Comment by mdonadoni** [comment]
> The CI complains about a link that should be fixed after we release reana 0.9.0

All good now!

**Pull request merged**